### DISCUSSION TITLE
Release Proposal for Apache Kvrocks 2.9.0

### DISCUSSION THREAD

**Discussion by PragmaTwice** [apache/kvrocks#2382] [General]
> Refer to https://kvrocks.apache.org/community/create-a-release

It's been more than 3 months since the last release ([v2.8.0](https://github.com/apache/kvrocks/releases/tag/v2.8.0)), and many exciting new features and fixes have been merged into the unstable branch. Now I would like to propose the next release version 2.9.0.

All commits on the unstable branch until 8547cfb48cb30dc3ad26214980e2c6a61c0e609a (included) will be considered as commits for this release, except:
- All RediSearch related (starts with `FT.`) commands will be disabled
- All stream group related commands will be disabled
- RocksDB version will be v8.11.4

The deadline for the cherry-picking period has been set as Jul 2, 2024, at 12:00 UTC+0. No new commits will be accepted after this time.

Feel free to leave your comments about this release proposal.

cc @apache/kvrocks-committers.

---

cherry-picked commits:
- ee69364df483badcb53bf0feac98391c18a2ffe3

**Comment by git-hulk** [comment]
+1 great to see this.

**Comment by torwig** [comment]
+1 from my side.

**Comment by Yangsx-1** [comment]
The stream consumer group commands will be finished in a few days, but it needs more tests. I believe we can release it in version 2.10.0 .

**Comment by mapleFU** [comment]
+1 for me

**Comment by PragmaTwice** [comment]
Cherry-picking period ended.

I'll start a release vote soon.
